docs/developer/attributes.mdx

```diff
@@ -0,0 +1,84 @@
+---
+title: Attributes
+sidebar_label: Attributes
+---
+
+## Introduction
+
+Setting up attributes will allow your users to filter your products catalogue.
+
+Attributes can be used for
+- [Products](developer/api-reference/objects/product.mdx) and [ProductVariants](developer/api-reference/objects/product-variant.mdx)
+- [Pages](developer/api-reference/objects/page.mdx)
```

Comment:
@IKarbowiak  working on attributes for categories.

Reply:
https://github.com/mirumee/saleor/tree/feature/category-attributes

Reply:
Can we sort those lists alphabetically?